### DISCUSSION TITLE
Bluetooth SCAN requires access to location

### DISCUSSION THREAD

**Discussion by Melynt3** [capacitor-community/bluetooth-le#631] [General]
Just wanted to ask if there's option to skip it? 

I have already definied it in the manifiest (testing in android) and app everysingle time asks for permissions to location, otherwise does not work.

**Comment by pwespi** [comment]
See https://github.com/capacitor-community/bluetooth-le?tab=readme-ov-file#optional-android-12-bluetooth-permissions
